portfolio/src/main/webapp/script.js

```diff
@@ -57,16 +57,23 @@ const getComments = function() {
     fetch('/data').then(response => response.json()).then((comments) => {
         const commentsContainer = document.getElementById('comments-container');
         commentsContainer.innerHTML = '';
-        
-        for(let i = 0; i < comments.length; i++) {
-            commentsContainer.appendChild(createListElement('Comment: ' + comments[i]));
-        }
+        comments.forEach((Comment) => {
+            commentsContainer.appendChild(createCommentElement(Comment));
+        });
     });
 }
 
 /** Creates an <li> element containing text. */
-const createListElement = function(text) {
-    const liElement = document.createElement('li');
-    liElement.innerText = text;
-    return liElement;
+function createCommentElement(comment) {
+    const commentElement = document.createElement('li');
+    commentElement.className = 'comment';
+    commentElement.innerText = comment.key.kind;
+
+    return commentElement;
+}
+
+const handleSubmitButton = function() {
```

Comment:
Clicking the submit button submits the form, which triggers a reload of the page, so this JS hardly has a chance to execute (and even if it did, it wouldn't get the new comment since that would require the reload to finish).